pkg/controllermanager/controller/seed/lifecycle/reconciler.go

```diff
@@ -132,8 +133,12 @@ func (r *Reconciler) Reconcile(ctx context.Context, req reconcile.Request) (reco
 		return reconcile.Result{RequeueAfter: r.Config.SyncPeriod.Duration}, nil
 	}
 
+	var lastTransitionTime time.Time
+	if conditionGardenletReady != nil {
+		lastTransitionTime = conditionGardenletReady.LastTransitionTime.UTC()
+	}
 	log.Info("Gardenlet has not sent heartbeats for at least the configured shoot monitor period, setting shoot conditions and constraints to 'Unknown' for all shoots on this seed",
-		"gardenletOfflineSince", conditionGardenletReady.LastTransitionTime.UTC(),
+		"gardenletOfflineSince", lastTransitionTime,
```

Comment:
I guess it makes more sense to log `gardenletOfflineSince`: `unknown` instead of `gardenletOfflineSince`: ` 0001-01-01 00:00:00 +0000 UTC.`. Don't we want to improve this aspect?

/hold

Reply:
Can we set the time to `unknown`?
What about setting it to seed creationTimestamp, if `GardenletReady` condition is not set?

Reply:
> Can we set the time to `unknown`?

We cannot set time.Time to unknown but for the logger it should matter - it accepts key-values as `interface{}`.

> What about setting it to seed creationTimestamp, if `GardenletReady` condition is not set?

Theoretically, `GardenletReady` condition could be missing not only on Seed creation. It can be removed by operator for some reason, or removed due to a potential bug on our side. I rather vote for `gardenletOfflineSince: unknown` or completely omitting logging the `gardenletOfflineSince` when the `GardenletReady` condition is not present - we simply don't know since when gardenlet is offline. WDYT?

Reply:
To make sure it logs the same thing, I prefer to keep `gardenletOfflineSince` with unknown. 
Made changes in 0b410ea43f46734d5e0230c9e69618370d5281a9. PTAL.